jaxrs-tck/src/main/java/ee/jakarta/tck/ws/rs/ee/rs/container/requestcontext/RequestFilter.java

```diff
@@ -149,7 +180,7 @@ public void getHeaders() {
     }
     abortWithEntity(sb.toString());
   }
-
+ 
```

Comment:
Excess blank.

Reply:
Normally I would remove this change, but all other methods appear to be separated by a blank line so I think it is appropriate to just leave this.

Reply:
This is no a blank line, it is a single blank before EOL, and I am used to remove those because some open source projects fail their QA in that case. :-)

Reply:
Thanks for explaining.   I'll fix this under #1248 since this PR has been merged.